src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala

```diff
@@ -854,14 +854,13 @@ class TezosDatabaseOperationsTest
     "get all values from the table with nulls as nones" in {
 
       val columns = List()
-
       val populateAndTest = for {
         _ <- Tables.Blocks ++= blocksTmp
         found <- sut.selectWithPredicates(Tables.Blocks.baseTableRow.tableName, columns, List.empty, List.empty, 3)
       } yield found
 
       val result = dbHandler.run(populateAndTest.transactionally).futureValue
-      result shouldBe List(
+      result should contain allElementsOf List(
```

Comment:
or `contain theSameElementsAs` ?

There's a subtle difference

Reply:
you are absolutely right! my bad, changed